indico/modules/events/contributions/templates/management/_contribution_list.html

```diff
@@ -0,0 +1,181 @@
+{% from 'message_box.html' import message_box %}
+
+{% macro render_contribution_list(event, contribs, sessions, tracks) %}
+    {% if not contribs %}
+        {%- call message_box('info') -%}
+            {% trans %}There are no contributions yet.{% endtrans %}
+        {%- endcall %}
+    {% else %}
+        <form method="POST">
+            <input type="hidden" name="csrf_token" value="{{ session.csrf_token }}">
+            <table class="i-table tablesorter">
+                <thead>
+                    <tr class="i-table">
+                        <th class="i-table checkbox-column"></th>
+                        <th class="i-table id-column">
+                            {% trans %}ID{% endtrans %}
+                        </th>
+                        <th class="i-table">
+                            {% trans %}Title{% endtrans %}
+                        </th>
+                        <th class="i-table">
+                            {% trans %}Schedule{% endtrans %}
+                        </th>
+                        <th class="i-table">
+                            {% trans %}Duration{% endtrans %}
+                        </th>
+                        <th class="i-table">
+                            {% trans %}People{% endtrans %}
+                        </th>
+                        <th class="i-table">
+                            {% trans %}Type{% endtrans %}
+                        </th>
+                        <th class="i-table">
+                            {% trans %}Session{% endtrans %}
+                        </th>
+                        <th class="i-table">
+                            {% trans %}Track{% endtrans %}
+                        </th>
```

Comment:
I think this should remove all empty lines:
```
{%- trans %}Sessions{% endtrans -%}
```
No need to add any `-` in the `else` nor `endif`